core/storage/statistics/gae_models.py

```diff
@@ -1134,6 +1134,25 @@ def delete_playthroughs_multi(cls, playthrough_ids):
         cls.delete_multi(instances)
 
 
+class LearnerAnswerDetailModel(base_models.BaseModel):
+    """Model for storing the approach/response user enters when he
+    is asked for it.
+    """
+    # ID of the entity.
+    entity_id = ndb.StringProperty(required=True, indexed=True)
+    # The type of entity, that whether it is state, question, or any other
+    # object if it is added in future.
+    entity_type = ndb.StringProperty(required=True, indexed=True)
+    # ID of the response.
+    response_id = ndb.StringProperty(required=True, indexed=False)
```

Comment:
Why isn't this indexed? What is this used for?

Reply:
Yeah I made it true. This can be used to uniquely identify any response.